src/SFA.DAS.ApprenticeFeedback.Application/Commands/CreateApprenticeFeedback/CreateApprenticeFeedbackCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MediatR;
+using SFA.DAS.ApprenticeFeedback.Domain.Models;
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.ApprenticeFeedback.Application.Commands.CreateApprenticeFeedback
+{
+    public class CreateApprenticeFeedbackCommand : IRequest<CreateApprenticeFeedbackResponse>
+    {
+        public Guid ApprenticeId { get; set; }
+        public long Ukprn { get; set; }
+        public OverallRating OverallRating { get; set; }
+        public List<FeedbackAttribute> FeedbackAttributes { get; set; }
+        public string ProviderName { get; set; }
```

Comment:
I think we can lose the following from the command:
- Ukprn
- ProviderName
- LarsCode
- StandardReference